bsp/src/org/jetbrains/bsp/project/test/environment/ConfigureBspTargetForWorksheet.scala

```diff
@@ -0,0 +1,38 @@
+package org.jetbrains.bsp.project.test.environment
+
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.module.Module
+import com.intellij.psi.PsiFile
+import javax.swing.Icon
+import org.jetbrains.bsp.BspBundle
+import org.jetbrains.bsp.BspUtil
+import org.jetbrains.bsp.Icons
+import org.jetbrains.bsp.project.test.environment.BspJvmEnvironment._
+import org.jetbrains.plugins.scala.actions.ScalaActionUtil
+import org.jetbrains.plugins.scala.lang.psi.api.ScalaFile
+import org.jetbrains.plugins.scala.worksheet.actions.topmenu.TopComponentAction
+import org.jetbrains.plugins.scala.worksheet.settings.WorksheetFileSettings
+
+class ConfigureBspTargetForWorksheet extends AnAction with TopComponentAction {
+  override def genericText: String = BspBundle.message("bsp.task.choose.target.title")
+
+  override def actionIcon: Icon = Icons.BSP_TARGET
+
+  override def acceptFile(file: ScalaFile): Boolean = {
+    val module = findModule(file)
+    BspUtil.isBspModule(module)
```

Comment:
I would say that it would be better not to show (even add) the icon for non-BSP projects at all, not just disable it.
![image](https://user-images.githubusercontent.com/3989292/80206713-891d2a00-8635-11ea-905f-9cb0c8a64359.png)

Reply:
Avoid adding is dificult, as there is no access to `project` during init of the component.  I updated PR to hide it on non BSP project.